src/Helpers/VulnerabilitiesHelper.js

```diff
@@ -11,16 +11,16 @@ export function createCveListByAccount(cveList) {
         let data = payload.data;
         let meta = payload.meta;
 
-        const descriptionMaxLength = 200;
-
         data = data.map(row => ({
             id: row.id,
             cells: [
                 <ImpactBadge impact={row.attributes.impact} hastooltip={true} key={row.id.toString()} />,
                 row.attributes.synopsis,
-                <CVERecordDescription
-                    description={row.attributes.description}
-                    maxstrlength={descriptionMaxLength}
+                <LongTextTooltip
+                    content={row.attributes.description}
+                    maxLength={200}
+                    tooltipPosition={'top'}
```

Comment:
```suggestion
```
This is default, isn't it?

Reply:
This values can be changed in the future. So it's better that we can use them also here.